src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
We did it in the past and there is no reason to change it in such a refactoring PR. 

By the way the NIST standard is only one among many others which require a space in front of a unit. 
For example the "The Cambridge Handbook for Editors" requires the same.

Interestingly there is actually a difference between
"50 s music" 
and 
"50s music"   
:-P